mini-e2e-helm.groovy

```diff
@@ -6,8 +6,10 @@ def git_repo = 'https://github.com/ceph/ceph-csi'
 def ref = "master"
 def namespace = 'cephcsi-e2e-' + UUID.randomUUID().toString().split('-')[-1]
 
+// ssh executes a given command on the reserved bare-metal machine
+// NOTE: do not pass " symbols on the command line, use ' only.
 def ssh(cmd) {
-	sh "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} '${cmd}'"
+	sh "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} \"${cmd}\""
```

Comment:
Do we need the double quotes on `${cmd}` ?

Reply:
I think we can get rid of them.

Reply:
We need some quotes around `${cmd}` because we want the whole command passed over ssh. Not execute a part over ssh and then run the rest after `&&` on the bare-metal system.